build.xml

```diff
@@ -234,11 +206,6 @@
     </target>
 
     <target
-        name="build-dev"
```

Comment:
Can you reinstate this and remove only the bit todo with phpunit and behat
Lines 237-241